### DISCUSSION TITLE
Anzeige der Software-Version via MQTT

### DISCUSSION THREAD

**Discussion by Sprudelverduenner** [jomjol/AI-on-the-edge-device#2909] [General]
Hallo Freunde,

gibt es eine Möglich per MQTT sich ein Topic mit der Version der installierten Firmware anzeigen zu lassen?
Ich benutze FHEM und hätte gerne dort ein dementsprechendes Reading...

Besten Dank vorab.
LG, Sprudelverduenner

**Comment by caco3** [comment]
Das habe ich gerade diese Woche implementiert: https://github.com/jomjol/AI-on-the-edge-device/pull/2891

Ist in rolling drin (und damit im nächsten Release)
